epsagon/generic_wrapper.go

```diff
@@ -172,8 +184,8 @@ func GoWrapper(config *Config, wrappedFunction interface{}, args ...string) Gene
 	}
 }
 
-// GoWrapper wraps the function with epsagon's tracer
-func ConcurrentGoWrapper(config *Config, wrappedFunction interface{}, args ...string) GenericFunction {
+// injectContextGoWrapper wraps the function with epsagon's tracer
+func injectContextGoWrapper(config *Config, wrappedFunction interface{}, args ...string) GenericFunction {
```

Comment:
How would we keep the option to use the ConcurrentGoWrapper now? (I see this function is not used)

Reply:
This is a mistake, will fix